doc/source/examples/cd_mmd_keops.ipynb

```diff
@@ -0,0 +1,513 @@
+{
```

Comment:
<p>Replace <code>&gt;&gt;</code> with <code>\gg</code> everywhere in the notebook for a better symbol of "much greater than".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/548/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>